src/react/account/details/properties/AccountKeys.tsx

```diff
@@ -133,125 +158,64 @@ function AccountKeys({ account }: Props) {
-          <T.Head>
-            {headerGroups.map((headerGroup, index) => (
-              <T.HeadRow key={index} {...headerGroup.getHeaderGroupProps()}>
-                {headerGroup.headers.map((column) => {
-                  const headerProps = column.getHeaderProps(
-                    column.getSortByToggleProps({
-                      title: '',
-                    }),
-                  );
-                  return (
-                    <T.HeadCell
-                      key={column.id}
-                      {...headerProps}
-                      style={{ ...column.headerStyle, ...headerProps.style }}
-                    >
-                      {column.render('Header')}
-                      <T.Icon>
-                        {!column.disableSortBy &&
-                          (column.isSorted ? (
-                            column.isSortedDesc ? (
-                              <i className="fas fa-sort-down" />
-                            ) : (
-                              <i className="fas fa-sort-up" />
-                            )
-                          ) : (
-                            <i className="fas fa-sort" />
-                          ))}
-                      </T.Icon>
-                    </T.HeadCell>
-                  );
-                })}
-              </T.HeadRow>
-            ))}
-          </T.Head>
-          <T.Body {...getTableBodyProps()}>
-            {rows.length > 0 ? (
-              rows.map((row) => {
-                prepareRow(row);
-                return (
-                  <T.Row isSelected={false} key={row.id} {...row.getRowProps()}>
-                    {row.cells.map((cell) => {
-                      const cellProps = cell.getCellProps();
-                      return (
-                        <T.Cell
-                          key={cell.id}
-                          {...cellProps}
-                          style={{
-                            ...cell.column.cellStyle,
-                            ...cellProps.style,
-                          }}
-                        >
-                          {cell.render('Cell')}
-                        </T.Cell>
-                      );
-                    })}
-                  </T.Row>
-                );
-              })
-            ) : (
-              <Warning
-                centered={true}
-                iconClass="fas fa-exclamation-triangle"
-                title="No key created"
-              />
-            )}
-          </T.Body>
-        </Table>
-      </T.Container>
-    </TableContainer>
+      <TableContainer>
+        {accessKeys.length > 0 ? (
+          <Table
+            columns={columns}
+            data={accessKeys}
+            defaultSortingKey="created_at"
+          >
+            <Table.SingleSelectableContent
+              rowHeight="h40"
+              separationLineVariant="backgroundLevel3"
+              backgroundVariant="backgroundLevel1"
+              onRowSelected={() => null}
```

Comment:
Likely if we don't need this we should make it optional in core ui if it is not already and drop the prop here

Reply:
`onRowSelected` is optional already. This is kind of a hack on my side (onRowSelected being a no-op and selectedId being defined but empty) to get the table to change the background of rows when hovering over them, but of course they're not actually selectable - nothing happens when you click on them. I did it because the previous table had that functionality, I'm okay to drop it if we don't need it. Or introduce a new prop `hoverable` which does only that. In some cases, when the action buttons are at the distant right end of the table and you're unsure which row you're targeting because there's a lot of empty space between the last column and action buttons, it's very useful to have that visual aid (for example I'm a bit astigmatic on the right eye and it affects me)